libnetwork/drivers/bridge/setup_ip_tables_linux_test.go

```diff
@@ -123,13 +129,20 @@ func assertChainConfig(d *driver, t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
+	if d.config.EnableIP6Tables {
+		d.natChainV6, d.filterChainV6, d.isolationChain1V6, d.isolationChain2V6, err = setupIPChains(d.config, iptables.IPv6)
+		if err != nil {
+			t.Fatal(err)
+		}
+	}
 }
 
 // Assert function which pushes chains based on bridge config parameters.
 func assertBridgeConfig(config *networkConfiguration, br *bridgeInterface, d *driver, t *testing.T) {
 	nw := bridgeNetwork{
-		portMapper: portmapper.New(""),
-		config:     config,
+		portMapper:   portmapper.New(""),
+		portMapperV6: portmapper.New(""),
+		config:       config,
```

Comment:
Interesting (not new in this PR, so something to look at separately), but it looks like we _never_ pass a value to `portmapper.New()`, so perhaps we should remove the argument. From that, also wondering if these should be initialised, or if we should make this struct have `nil` if not used (and only initialise a portmapper for those IP-versions that we use) 🤔 

<img width="880" alt="Screenshot 2023-09-11 at 13 09 36" src="https://github.com/moby/moby/assets/1804568/67218d1e-2199-4b66-a169-925eca2fca08">

Reply:
I'll take a look and include any worthwhile cleanups in a separate PR.